### PR TITLE
Expose new API to download pre-generated vpn profile package URL

### PR DESCRIPTION
**Pull request by ritwikbasu**
Expose new API to download pre-generated vpn profile package URL

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by olydis** [comment]
@ravbhatnagar New operation.

**Comment by ritwikbasu** [comment]
@ravbhatnagar - Can you please approve this PR or let me know if you have any concerns? This is one of our Ignite deliverables and we need this change to go in before proceeding with SDK/PS coding.

**Comment by ravbhatnagar** [comment]
@ritwikbasu - this API wont work. Please schedule a 15 min sync with me on Wednesday to go over why.

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/virtualNetworkGateway.json
Comment (ravbhatnagar) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/virtualNetworkGateway.json]: Please define the model  for the response instead of just marking it as string. Since the change has already gone in on the service side, please consider it in the next api version rev. Not blocking on it for now.

**Review by ravbhatnagar — APPROVED** [review]
Just the one note that i added which NRP will consider fixing in the next api-version rev.

**Comment by ravbhatnagar** [comment]
@olydis - ARM signed off.

**Review by olydis — APPROVED** [review]

**Pull request merged**
